dotcom-rendering/src/server/prod-server.ts

```diff
@@ -60,6 +61,7 @@ export const prodServer = (): void => {
 	app.post('/KeyEvents', logRenderTime, handleKeyEvents);
 	app.post('/Front', logRenderTime, handleFront);
 	app.post('/FrontJSON', logRenderTime, handleFrontJson);
+	app.post('/apps/Article', logRenderTime, handleAppsArticle);
```

Comment:
Could we be consistent with the paths, directories and function names we're using for different products? So either:

```
/WebArticle -> handleWebArticle
/AMPArticle -> handleAMPArticle
/AppsArticle -> handleAppsArticle
/EditionsArticle -> handleEditionsArticle
```

or

```
/web/Article -> handleWebArticle
/amp/Article -> handleAMPArticle
/apps/Article -> handleAppsArticle
/editions/Article -> handleEditionsArticle
```

Reply:
good suggestion, I'd prefer `/AppsArticle`